public/css/main.less

```diff
@@ -526,7 +526,8 @@ thead {
 }
 
 form.code {
-  font-size: 22px;
+  font-size: 16px;
```

Comment:
We picked 22px font size as that makes an almost perfect 80 character width editor area. Will 16px detract from readability?